package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "kyodo-protocol",
+  "name": "kyodo-protocol-frontend",
```

Comment:
Since the repo name is kyodo-protocol, I think we should keep this package.json named the same since it is in the root directory. Or, we could create a folder named "dapp" and put all the front files in it, and leave the root directory with another package.json file that compiles and builds everything. But for now, I would keep this name with `kyodo-protocol`

Reply:
you're right, I fixed that